### ISSUE TITLE
Can't get it to work locally

### ISSUE THREAD

**Issue by amcsi** [rubenarakelyan/monzo-qif-exporter#4]
I tried cloning the project, doing `composer install` within the `protected/` folder (which isn't mentioned in the readme by the way)

Then in CLI in the `public/` folder I did `php -S 0.0.0.0:9999 ./index.php`, but when I go to localhost:9999 in my browser, I get "Whoops, looks like something went wrong." twice without any details.

What am I doing wrong?

**Comment by rubenarakelyan** [comment]
I don't think you can run a Laravel app like this, since the `index.php` is handled as a "router" script and all requests (even for files that exist in the filesystem) are routed to it, whereas the `.htaccess` file handles this and only directs requests for non-existent files to `index.php`.

**Comment by amcsi** [comment]
@rubenarakelyan It should be possible to emulate this most typical .htaccess use case with PHP's built-in webserver. You'd tell it where the public folder is, and also which script all requests should be routed to if the file doesn't exist in the public folder.

**Comment by rubenarakelyan** [comment]
OK, is there anything in the `laravel.log` file that sheds more light on the actual error?
